.github/workflows/test-ui.yml

```diff
@@ -47,9 +47,12 @@ jobs:
 
       - name: Run UI tests
         run: >
-          npm run test:pre-init &&
-          npm run test:init &&
-          npm run test:post-init
+          NODE_PATH=$(pwd)/src/ui/node_modules
+          npm --prefix src/ui run test:pre-init &&
+          NODE_PATH=$(pwd)/src/ui/node_modules
+          npm --prefix src/ui run test:init &&
+          NODE_PATH=$(pwd)/src/ui/node_modules
+          npm --prefix src/ui run test:post-init
```

Comment:
Do you have to keep setting the same NODE_PATH over and over again?

Reply:
no, ill change it to an `export` beforehand